code/sheet_cleaner/sheet_processor.py

```diff
@@ -163,11 +163,11 @@ def push_to_github(self):
         # Create script for uploading to github
         script  = 'set -e\n'
         script += 'cd {}\n'.format(self.git_repo_path)
-        script += 'git pull origin master\n'
         
         for g in self.for_github:
             script += f'git add {g}\n'
         script += 'git commit -m "data update"\n'
+        script += 'git pull origin master\n'
```

Comment:
git pull doesn't reset whatever you had committed afaik.
https://stackoverflow.com/questions/18529206/when-do-i-need-to-do-git-pull-before-or-after-git-add-git-commit seems to say the same thing.

I'd actually just remove that pull command, the script is run from a freshly pulled version every time it runs so this is useless.

Reply:
That makes sense. The original pull was before the commit - after the commit I think would have been fine. I agree that removing the pull command is a great solution.